contracts/nois-drand/src/contract.rs

```diff
@@ -51,13 +52,23 @@ pub fn instantiate(
         incentive_denom: msg.incentive_denom,
     };
     CONFIG.save(deps.storage, &config)?;
+    set_contract_version(
+        deps.storage,
+        env!("CARGO_PKG_NAME"),
+        env!("CARGO_PKG_VERSION"),
+    )?;
     Ok(Response::default())
 }
 
 // This no-op migrate implementation allows us to upgrade within the 0.7 series.
 // No state changes expected.
 #[cfg_attr(not(feature = "library"), ::cosmwasm_std::entry_point)]
-pub fn migrate(_deps: DepsMut, _env: Env, _msg: Empty) -> StdResult<Response> {
+pub fn migrate(deps: DepsMut, _env: Env, _msg: Empty) -> StdResult<Response> {
+    set_contract_version(
+        deps.storage,
+        env!("CARGO_PKG_NAME"),
```

Comment:
This just uses e.g. "nois-drand" from Cargo.toml. However, cw2 seems to expect a crates.io prefix:

```rust
    /// contract is a globally unique identifier for the contract.
    /// it should build off standard namespacing for whichever language it is in,
    /// and prefix it with the registry we use.
    /// For rust we prefix with `crates.io:`, to give us eg. `crates.io:cw20-base`
```

I wonder if this prefix should also be used if:
- the packages is not on crates.io
- anyone else can claim the name on crates.io